SassHandler.php

```diff
@@ -203,15 +203,14 @@ class SassHandler extends CApplicationComponent
      */
     public function init()
     {
-        $vendorPath = __DIR__ . '/../../';

```

Comment:
Sorry if i miss something. From what i understand, is that the composer install the files inside the vendor sub-directory of protected and not directly on protected. So, for this reason i think that the vendor needs to go on the $vendorPath variable.